addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRuleUnitTest.java

```diff
@@ -697,6 +697,40 @@ void shouldReturnExpectedMappings() {
                 is(equalTo(CommonAlertTag.WSTG_V42_CONF_05_ENUMERATE_INFRASTRUCTURE.getValue())));
     }
 
+    @Test
+    void shouldReturnExpectedExampleAlert() {
+        // Given / When
+        List<Alert> alerts = rule.getExampleAlerts();
+        Alert alert = alerts.get(0);
+        Map<String, String> tags = alert.getTags();
+        // Then
+        assertThat(alerts.size(), is(equalTo(1)));
+        assertThat(alert.getTags().size(), is(equalTo(4)));
+        assertThat(
+                alert.getTags().containsKey(CommonAlertTag.CUSTOM_PAYLOADS.getTag()),
+                is(equalTo(true)));
```

Comment:
I know this is what was done in all other places, but do we really need to check the value? Could just do e.g. `assertThat(alert.getTags(), hasKey(CommonAlertTag.CUSTOM_PAYLOADS.getTag()))`.
Something for another PR in any case.

Reply:
I'm fine with simplifying it. I'll tackle it over the weekend.

Reply:
`        assertThat(tags, hasKey(CommonAlertTag.CUSTOM_PAYLOADS.getTag()));`